index.html

```diff
@@ -1,14 +1,20 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>ToDoApp</title>
-</head>
-<body>
-    
-
+  </head>
+  <body>
+    <h1>ToDoApp</h1>
```

Comment:
No something mandatory, but It's always a good practice to use Semantic HTML tags to improve readability and structure. For example we might wrap this h1 into a <header>

```
    <header>
        <h1>ToDoApp</h1>
    </header>
```